singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -647,6 +651,491 @@ class PTESolverRhoT : public mix_impl::PTESolverBase<EOSIndexer, RealIndexer> {
   Real Tequil, Ttemp;
 };
 
+// fixed temperature solver
+inline int PTESolverFixedTRequiredScratch(const int nmat) {
+  int neq = nmat;
+  return neq * neq                 // jacobian
+         + 4 * neq                 // dx, residual, and sol_scratch
+         + 2 * nmat                // rhobar and u in base
+         + 2 * nmat                // nmat sized arrays in fixed T solver
+         + MAX_NUM_LAMBDAS * nmat; // the cache
+}
+inline size_t PTESolverFixedTRequiredScratchInBytes(const int nmat) {
+  return PTESolverFixedTRequiredScratch(nmat) * sizeof(Real);
+}
+
+template <typename EOSIndexer, typename RealIndexer, typename LambdaIndexer>
+class PTESolverFixedT : public mix_impl::PTESolverBase<EOSIndexer, RealIndexer> {
```

Comment:
I'm torn on the name `PTESolverFixedT`. But I can't think of a better name.

Reply:
resolving for now.